packages/transactions/src/coders/output.ts

```diff
@@ -108,48 +107,6 @@ export class OutputContractCoder extends Coder<OutputContract, OutputContract> {
   }
 }
 
-export type OutputMessage = {
```

Comment:
`OutputMessage` is redundant now. We don't need to specify it in the outputs now. Also removed it from the `OutputType` enum